<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;UD Digi Kyokasho NK-R&quot;;</string>
   </property>
   <property name="text">
    <string>Query Tools</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;UD Digi Kyokasho NK-R&quot;;</string>
   </property>
   <property name="text">
    <string>Query Keywords</string>
   </property>
  </widget>
  <widget class="QComboBox" name="query_tool">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;UD Digi Kyokasho NK-R&quot;;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="query_keyword">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;UD Digi Kyokasho NK-R&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;UD Digi Kyokasho NK-R&quot;;
color: rgb(47, 118, 231)</string>
   </property>
   <property name="text">
    <string>Clinical Management system</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>570</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;UD Digi Kyokasho NK-R&quot;;
color: rgb(47, 118, 231)</string>
   </property>
   <property name="text">
    <string>Query Results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;UD Digi Kyokasho NK-R&quot;;</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="show_query">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>620</y>
     <width>821</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;UD Digi Kyokasho NK-R&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tool_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;UD Digi Kyokasho NK-R&quot;;</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyword_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>931</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;UD Digi Kyokasho NK-R&quot;;
color: rgb(255, 65, 84);</string>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="tool_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>931</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;UD Digi Kyokasho NK-R&quot;;
color: rgb(255, 65, 84);</string>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>580</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;UD Digi Kyokasho NK-R&quot;;</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
